gallery/how_to/compile_models/from_keras.py

```diff
@@ -102,8 +102,8 @@
 shape_dict = {"input_1": data.shape}
 mod, params = relay.frontend.from_keras(keras_resnet50, shape_dict)
 # compile the model
-target = "cuda"
-dev = tvm.cuda(0)
+target = "llvm"
+dev = tvm.cpu(0)
```

Comment:
I know that colab provides several runtimes (`CPU` or `GPU`), which can be selected manually. Can we automatically select preferable runtime for some scripts, e.g. where we want to use `GPU`?